EEBOS/SCROWL
# [SC-262] - Fix Makefile

### Short summary

- Minor update to the makefile to copy over the node_modules from the container to the host machine so VS Code extensions and linting work. 
- Added a requested feature to get any secret files when `startOver=true` is used
-

---

### Test steps

`make cleanStart`

This command mainly focused on two scenarios:
- A new member starting the project from scratch
   - It will fetch secret files
   - Build the docker image 
   - Start the project
- An existing member that just want to have a clean start
   - I does remove any exiting node_modules and yarn.lock files for you
   - It will fetch secret files
   - Build the docker image (Without using caching so it truly builds from scratch) 
   - Start the project     

NOTE: Since we are building the image locally this command will quite some time.
 
---

`make start startOver=true`

There was a request to fetch secret files as well. So now you should see in the terminal that the `getLatestSecrets` target is used before starting the project.
